scripts/nayduck.py

```diff
@@ -182,11 +182,16 @@ def run_locally(tests):
         # See nayduck specs at https://github.com/near/nayduck/blob/master/lib/testspec.py
         fields = test.split()
 
+        timeout = ""
```

Comment:
I guess using `timeout` tool is one way to do it. But if we do that we need `--verbose` argument to `timeout`. The way I would have done it is to initialise `timeout` with `None`, parse it as int and then use `subprocess.check_call(…, timeout=timeout)`.

Reply:
` subprocess.check_call` would be better, but then we would need some way of parsing time for example `2h` to seconds.

Reply:
You could copy the code from nayduck.

```
def _parse_timeout(timeout: str) -> int:
    """Parses timeout interval and converts it into number of seconds.

    Args:
        timeout: An integer with an optional ‘h’, ‘m’ or ‘s’ suffix which
            multiply the integer by 3600, 60 and 1 respectively.
    Returns:
        Interval in seconds.
    """
    if timeout and (mul := {'h': 3600, 'm': 60, 's': 1}.get(timeout[-1])):
        timeout = timeout[:-1]
    else:
        mul = 1
    return int(timeout) * mul
```

Reply:
@mina86 Thanks, I added it to PR.